ReadMe.md

```diff
@@ -3,4 +3,7 @@
 Learning how to use github properly
 
 20-01-2023
-16:30
\ No newline at end of file
+16:30
```

Comment:
The time should reflect the the moment that the git push command was pressed (example)